src/style/mod.rs

```diff
@@ -497,6 +497,6 @@ mod tests {
         assert_type_size::<Line<GridPlacement>>(8);
 
         // Overall
-        assert_type_size::<Style>(352);
+        assert_type_size::<Style>(552);
```

Comment:
This is not ideal! But we're probably going to have to deal with it at some point (some mitigations are potentially available).